.github/workflows/solutions-dpg-qa.yaml

```
@@ -0,0 +1,156 @@
+          id: install
+
+      - name: Creating namespace
+        run: kubectl create namespace egov
+        continue-on-error: true
+
+      - name: Verify kubectl get nodes
+        run: |
+          kubectl get nodes
+
+      - name: Install sops
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:3:10: Double quote to prevent globbing and word splitting

(shellcheck)